src/CustomFunction.ts

```diff
@@ -28,115 +31,78 @@ interface FunctionFailArguments {
 
 export type FunctionFailFn = (params: FunctionFailArguments) => Promise<FunctionsCompleteErrorResponse>;
 
-export interface CustomFunctionExecuteMiddlewareArgs extends SlackEventMiddlewareArgs<'function_executed'> {
+export type SlackCustomFunctionMiddlewareArgsOptions = { autoAcknowledge: boolean };
```

Comment:
This is the same type as `SlackEventMiddlewareArgsOptions`; do we plan on extending it beyond this? If not, could put this under `src/types/utilities` or `src/types/middleware` or some other such central place and re-use in both locations.

Reply:
Good catch 💯 I've improved this to be a more general `isSlackEventMiddlewareArgsOptions`